course/admin.py

```diff
@@ -39,7 +39,14 @@ class TokenValueAdmin(admin.ModelAdmin):
 
 class UserQuestionJunctionAdmin(admin.ModelAdmin):
     list_filter = ('user__username', 'question')
-    list_display = ('user', 'question', 'is_solved', 'is_partially_solved',)
+    list_display = ('user', 'question', 'is_solved', 'is_partially_solved', 'is_favourite')
+
+
+class ReportQuestionAdmin(admin.ModelAdmin):
+    list_filter = ('user__username', 'question')
+    list_display = ('user', 'question', 'report_timestamp', 'unclear_description', 'test_case_incorrect_answer',
```

Comment:
We don't need to display all the fields in the admin list.

Reply:
Moved the QuestionReportAdmin to be under general > admin.py now.